BuildToolsCliVersion.txt

```diff
@@ -0,0 +1 @@
+1.0.0-preview2-002733
```

Comment:
Yes unfortunately we temporarily need the duplicated shared framework as BuildTools doesn't have the necessary extensibility points to use an arbitrary shared framework/CLI. 

@sokket please file an issue on BuildTools to track this.